### PR TITLE
[New practice exercise] Pop count

### PR DESCRIPTION
**Pull request by Cool-Katt**
Adding pop-count exercise to the js track

### CODE REVIEW

File: exercises/practice/pop-count/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/node_modules
+/bin/configlet
```
Comment (IsaacG) [inline exercises/practice/pop-count/.gitignore:2]: Was this file auto-created? It looks ... wrong. These ought to be in the root ignore file.

File: exercises/practice/pop-count/.npmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+audit=false
```
Comment (IsaacG) [inline exercises/practice/pop-count/.npmrc:1]: This is also defined in the root and I suspect isn't needed

File: exercises/practice/pop-count/LICENSE [hunk 3]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (IsaacG) [inline exercises/practice/pop-count/LICENSE:1]: Licensing is done on a repo level, not an exercise level.

**Review by IsaacG — COMMENTED** [review]
I left comments that may very well be outside the scope of this PR.

There are some checks which are failing.

File: exercises/practice/pop-count/.gitignore (hunk 1 above)
Comment (Cool-Katt) [reply exercises/practice/pop-count/.gitignore:2]: This was an auto-generated file.

File: exercises/practice/pop-count/.npmrc (hunk 2 above)
Comment (Cool-Katt) [reply exercises/practice/pop-count/.npmrc:1]: Another auto-generated file.

File: exercises/practice/pop-count/LICENSE (hunk 3 above)
Comment (Cool-Katt) [reply exercises/practice/pop-count/LICENSE:1]: yet another auto-generated file.

**Comment by Cool-Katt** [comment]
All of the issues that @IsaacG pointed to are with auto-generated files. The command used to get those was `ASSIGNMENT=pop-count npx babel-node scripts/sync` I think, from the `CONTRIBUTING.md`. I've not messed with those, as i don't know if they're actually needed.

I also ran through the CI checks and it was all formatting errors. Should be all cleared after the last commit, if you can run them again.

**Review by IsaacG — APPROVED** [review]

File: exercises/practice/pop-count/package.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@exercism/javascript-pop-count",
+  "description": "Exercism practice exercise on pop-count",
+  "author": "Katrina Owen",
+  "contributors": [
+    "Derk-Jan Karrenbeld <derk-jan+git@karrenbeld.info> (https://derk-jan.com)",
+    "Tejas Bubane (https://tejasbubane.github.io/)"
+  ],
```
Comment (IsaacG) [inline exercises/practice/pop-count/package.json:8]: @Cool-Katt  Do you want to update these?

**Review by IsaacG — APPROVED** [review]

File: exercises/practice/pop-count/package.json (hunk 4 above)
Comment (Cool-Katt) [reply exercises/practice/pop-count/package.json:8]: You mean to add myself as a contributor? I don't know if I properly understand what you mean.

Comment (IsaacG) [reply exercises/practice/pop-count/package.json:8]: Yup. Or as the author. The only person that worked on this exercise is you so I think it ought to be reflected in the author/contributors docs.

**Review by IsaacG — APPROVED** [review]

**Review by ErikSchierboom — APPROVED** [review]
Thanks

**Pull request merged**